### PR TITLE
feat: add scaffold command to monopod

### PR DESCRIPTION
**Pull request by stormqueen1990**
### Changes

* Create a new `scaffold` command in `monopod` to aid creation of new images.
* Reorganize the `images/TEMPLATE` folder to a flat structure to make it easier to read the template files.
* Change templates to work with Go `text/templates`.

### CODE REVIEW

File: images/TEMPLATE/main-config.tf [hunk 1]
```diff
@@ -12,7 +12,7 @@ variable "extra_packages" {
 }
```
Comment (pdeslaur) [inline images/TEMPLATE/main-config.tf:12]: Should `TODO YOUR PACKAGE HERE` be replaced with `.PackageName`? If so, we could also update the `apko` YAML to include the comment:

`# {{ .PackageName }} comes in via var.extra_packages`

**Review by priyawadhwa — COMMENTED** [review]
This LGTM! Maybe we could add a test to generate a sample scaffold & compare to the expected output.

**Review by vaikas — COMMENTED** [review]
Thanks for doing this!

File: images/TEMPLATE/main-config.tf (hunk 1 above)
Comment (stormqueen1990) [reply images/TEMPLATE/main-config.tf]: Yes that makes sense to me!

Comment (stormqueen1990) [reply images/TEMPLATE/main-config.tf]: Done!

**Comment by stormqueen1990** [comment]
> This LGTM! Maybe we could add a test to generate a sample scaffold & compare to the expected output.

Yes! That's a great idea, I'll add it.

**Review by imjasonh — APPROVED** [review]

**Pull request merged**